google/cloud/pubsub/internal/tracing_pull_ack_handler.cc

```diff
@@ -30,11 +32,43 @@ namespace pubsub_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 #ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
-
 using Attributes =
     std::vector<std::pair<opentelemetry::nostd::string_view,
                           opentelemetry::common::AttributeValue>>;
+using Links =
+    std::vector<std::pair<opentelemetry::trace::SpanContext, Attributes>>;
+
+namespace {
+Links CreateLinks() {
```

Comment:
Consider:

```suggestion

namespace {

auto CreateLinks() {
  using Links =
      std::vector<std::pair<opentelemetry::trace::SpanContext, Attributes>>;
```

It avoids polluting the TU's namespace.